src/CTA.Rules.Actions/ActionHelpers/ConfigMigrate.cs

```diff
@@ -56,7 +56,7 @@ private configuration ProcessWebConfig(string projectDir)
                         webConfig = (configuration)ser.Deserialize(reader);
                     }
                 }
-                catch (Exception ex)
```

Comment:
LogError accepts an exception parameter. can we change to that instead. something like:
LogHelper.LogError(ex, string.Format("Error processing web.config file {0}", webConfigFileDir))